src/components/calcite-dropdown/calcite-dropdown.e2e.ts

```diff
@@ -857,6 +857,78 @@ describe("calcite-dropdown", () => {
     expect(await dropdownWrapper.isVisible()).toBe(false);
   });
 
+  it("correct role and aria properties are applied based on selection type", async () => {
+    const page = await newE2EPage();
+    await page.setContent(`
+    <calcite-dropdown>
```

Comment:
@jcfranco can we maybe run prettier pre commit?

Reply:
Prettier runs on each commit, but it doesn't format embedded content if it can't recognize it. [Looking at the docs](https://prettier.io/docs/en/options.html#embedded-language-formatting), I think we could have this work automatically wrapping `dedent` in a tagged template called `html`, which Prettier will pick up. Then we would only need to do something similar to get automatic formatting:

```ts
const page = await newE2EPage({
  html: html`
      ...some markup
  `
});
```

I'll submit a PR for this. Thanks for the idea! 🎉